### PR TITLE
Develop/database fix test

### PR DESCRIPTION
**Pull request by KanaiYuma-aist**
fix unit, codestyle, integration, resumption test for Develop/database branch

### CODE REVIEW

File: aiaccel/util/process.py [hunk 1]
```diff
@@ -128,6 +130,34 @@ def ps2joblist() -> List[dict]:
...
+    Returns:
+        List[dict]: A job list of ps result.
+
+    Raises:
+        KeyError: Causes when required keys are not contained in a ps result.
+    """
+
+    job_list = []
+
+    for pInfoDict in psutil.process_iter(['pid', 'username', 'status', 'create_time', 'cmdline']):
```
Comment (aramoto99) [inline aiaccel/util/process.py:148]: Might it be better to lowercase it to match the rest of the description?
pInfoDict -> p_info

**Comment by KanaiYuma-aist** [comment]
https://github.com/aistairc/aiaccel/pull/18#issuecomment-1189921856

add comment to https://github.com/aistairc/aiaccel/pull/18#issue-1310556168

**Review by aramoto99 — CHANGES_REQUESTED** [review]

File: aiaccel/storage/hp/fs.py [hunk 2]
```diff
@@ -127,4 +126,5 @@ def all_delete(self) -> None:
         """
         self.update()
         for hp in self.datas.data:
-            hp.remove()
+            if hp is not None:
```
[the remark this answers is not in the document]
Comment (KanaiYuma-aist) [reply aiaccel/storage/hp/fs.py:129]: When running tests in `tests/unit/storage/fs/test_fs_hp.py`, `storage.hp.set_any_trial_param` would store None in the array and an error would occur trying to execute `(None).remove()`. 
So add `if hp is not None:`.

---

tests/unit/storage/fs/test_fs_hp.py のテスト実行時に
`storage.hp.set_any_trial_param` によって配列に None が格納され、
`(None).remove()` を実行しようとしてエラーが発生していたため、
`if hp is not None:` を追加しました。

https://github.com/aistairc/aiaccel/blob/7c0a3aaeecc8f8be3c87a80a5704d6f3f950e115/tests/unit/storage/fs/test_fs_hp.py#L97-L115

[the remark this answers is not in the document]
Comment (yoshipon) [reply aiaccel/storage/hp/fs.py:129]: So, is it an intended behavior that the items in `self.datas.data` can become `None`...?

[the remark this answers is not in the document]
Comment (KanaiYuma-aist) [reply aiaccel/storage/hp/fs.py:129]: > (Perhaps the next two items need to be corrected as well).

I changed trial_id to 3 in the test code `tests/unit/storage/fs/test_fs_timestamp.py` `tests/unit/storage/fs/test_fs_error.py` which calls those processes, and the same error occurred.

So I think it is necessary to add `if hp is not None:` as well.

---

> （以下のコードにも似たような処理があります．変更すべきですか？）

試しにそれらの処理を呼び出しているテストコード
`tests/unit/storage/fs/test_fs_timestamp.py`
`tests/unit/storage/fs/test_fs_error.py`
の trial_id に当たる部分を 3 などに変更してテストした所、
同様のエラーが発生しました。

なので、同様に`if hp is not None:`の追加が必要だと思われます。

**Review by aramoto99 — APPROVED** [review]

**Pull request merged**